.github/workflows/ci.yml

```diff
@@ -42,7 +42,7 @@ jobs:
         os:
           - 'centos-7'
           - 'centos-8'
-          - 'fedora-31'
+          - 'fedora-32'
```

Comment:
Before `fedora-latest` gets mentioned, the COPR repo doesn't have packages for Fedora 33 yet so it breaks testing.